<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>726</width>
    <height>635</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DisCODe</string>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowNestedDocks|QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>350</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>446</width>
         <height>550</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>726</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTask">
    <property name="title">
     <string>Task</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="actionUnload"/>
   </widget>
   <addaction name="menuTask"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConnect"/>
   <addaction name="actionFinish"/>
   <addaction name="separator"/>
   <addaction name="actionStart_all"/>
   <addaction name="actionStop_all"/>
   <addaction name="separator"/>
   <addaction name="actionRefresh"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="wComponents">
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Components</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QListWidget" name="listComponents"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="wSubtasks">
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Subtasks</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QListWidget" name="listSubtasks"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionStart_all">
   <property name="icon">
    <iconset resource="gui.qrc">
     <normaloff>:/icons/play</normaloff>:/icons/play</iconset>
   </property>
   <property name="text">
    <string>Start all</string>
   </property>
  </action>
  <action name="actionStop_all">
   <property name="icon">
    <iconset resource="gui.qrc">
     <normaloff>:/icons/stop</normaloff>:/icons/stop</iconset>
   </property>
   <property name="text">
    <string>Stop all</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionUnload">
   <property name="text">
    <string>Unload</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="icon">
    <iconset resource="gui.qrc">
     <normaloff>:/icons/refresh</normaloff>:/icons/refresh</iconset>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="icon">
    <iconset resource="gui.qrc">
     <normaloff>:/icons/connect</normaloff>:/icons/connect</iconset>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
   <property name="toolTip">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionFinish">
   <property name="icon">
    <iconset resource="gui.qrc">
     <normaloff>:/icons/exit</normaloff>:/icons/exit</iconset>
   </property>
   <property name="text">
    <string>Finish</string>
   </property>
   <property name="toolTip">
    <string>Stop DisCODe execution</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="gui.qrc"/>
 </resources>
 <connections/>
</ui>
